modules/client-configuration/nav-client-configuration-guide.adoc

```diff
@@ -87,14 +87,13 @@ endif::[]
 // Delete clients
 ** xref:delete-clients.adoc[Delete Clients]
 // Automated Installation
-** xref:autoinst-intro.adoc[Automating Client Installation]
-*** xref:autoinst-setup.adoc[Autoinstallation Files and Distributions]
+** xref:autoinst-intro.adoc[Automating Operating System Installation]
```

Comment:
I will even shorten by removing "Automated"...

Inside the chapter, it appears as "Autoinstallation", which is more cryptic, but kind of sounding nice

Reply:
I was just adjusting these so they are declarative noun phrases rather than verb phrases: https://github.com/uyuni-project/uyuni-docs/wiki/wug#headings